.travis.yml

```diff
@@ -43,7 +43,7 @@ env:
   - CAMLP5_VER=".6.14"
   - CAMLP5_VER_BE=".7.03"
   - FINDLIB_VER=".1.4.1"
-  - FINDLIB_VER_BE=".1.7.3"
+  - FINDLIB_VER_BE=".1.8.0"
```

Comment:
Why are you changing this in this PR?
It breaks travis btw.

Reply:
1.8.0 is installed by the docker image, so this is in preparation of switching Travis to it.